CMakeLists.txt

```diff
@@ -65,6 +65,16 @@ elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
     set(ARCH_NAME "X86")
   endif ()
 endif ()
+if ("${CMAKE_SYSTEM_NAME}" MATCHES "Darwin")
+  set(GENERATED_MACH_SOURCES
+      ${CMAKE_CURRENT_BINARY_DIR}/mach_excServer.c
+      ${CMAKE_CURRENT_BINARY_DIR}/mach_excUser.c
+  )
+  add_custom_command(OUTPUT ${GENERATED_MACH_SOURCES}
+    COMMAND mig ${CMAKE_SOURCE_DIR}/Sources/Host/Darwin/mach_gen.defs
```

Comment:
nit: You should name these files something like `macGen.defs` instead of mach_gen.defs`.